malcolm/modules/ca/parts/cadoublearraypart.py

```diff
@@ -19,13 +19,24 @@ def __init__(self,
                  widget=None,  # type: util.AWidget
                  group=None,  # type: util.AGroup
                  config=True,  # type: util.AConfig
+                 display_from_pv=False,  # type: util.AGetLimits
                  ):
         # type: (...) -> None
         super(CADoubleArrayPart, self).__init__(name)
+        self.display_from_pv = display_from_pv
+
         self.caa = util.CAAttribute(
-            NumberArrayMeta("float64", description), util.catools.DBR_DOUBLE,
+            NumberArrayMeta("float64", description, display=Display()), util.catools.DBR_DOUBLE,
```

Comment:
```suggestion
            NumberArrayMeta("float64", description), util.catools.DBR_DOUBLE,
```
Don't need this as the code underneath does it for us